etna/lib/etna/environment_scoped.rb

```diff
@@ -0,0 +1,19 @@
+class EnvironmentScoped < Module
```

Comment:
This makes me think we ought to move over this logic from polyphemus and have polyphemus import it from etna.  But we can do that in future PR.

Reply:
DERP NEVERMIND